packages/react/src/components/Swiper/Swiper.tsx

```diff
@@ -115,6 +116,7 @@ const SwiperContainer = styled('div', {
 
   '& > *': {
     scrollSnapAlign: ownerState.alignment,
+    scrollSnapStop: ownerState.snapStop === 'always' ? 'always' : 'normal',
```

Comment:
Можно ему задать дефолтное значение `normal` и тогда
```suggestion
    scrollSnapStop: ownerState.snapStop,
```